apps/rush-lib/src/api/RushLastInstallFlag.ts

```diff
@@ -0,0 +1,12 @@
+import { LastInstallFlag } from './LastInstallFlag';
+import { RushConfiguration } from './RushConfiguration';
+
+export class RushLastInstallFlag extends LastInstallFlag {
```

Comment:
>RushLastInstallFlag [](start = 13, length = 19)

Having a helper makes sense.  Modeling the helper as a subclass (the most brittle form of relationship) doesn't seem best.

Reply:
Might be better/easier to make this a static initializer.

---
In reply to: [208431171](https://github.com/Microsoft/web-build-tools/pull/766#discussion_r208431171) [](ancestors = 208431171)